### ISSUE TITLE
Mount controllers to different endpoints

### ISSUE THREAD

**Issue by alexproca** [Romakita/ts-express-decorators#13]
Right now we can add all controller under same endpoint

```
this.setEndpoint('/rest')
            .scan(appPath + "/controllers/**/**.js")
            .createHttpServer(8000)
            ;
```

I would be nice to have the ability to mount various directories under various endpoints.

```
this.mount('/rest', appPath + "/controllers/**/**.js")
    .mount('/rest/v1', appPath + "/v1/controllers/**/**.js")
            .createHttpServer(8000)
            ;
```

**Comment by Romakita** [comment]
I think it's even necessary to do it.

**Comment by vincent178** [comment]
I like this idea 👍

**Comment by Romakita** [comment]
This feature is available with version 1.3.0-6 ;)
